### PR TITLE
Removes vampires and shadowlings from halloween

### PR DESCRIPTION
**Pull request by thehighwayman**
### Intent of your Pull Request
Raptor thought these were the worst so they're gone now
#### Changelog

:cl:  
rscdel: Bye halloween fun 
/:cl:

### CODE REVIEW

**Comment by GuppyLaxx** [comment]
this is terrible i am so sorry but my adding lines you're not only making it NOT modular and this could also just be done 100 times more modular by just, making a new file and just doing
``` 
/datum/species/shadow/check_roundstart_eligible()
          return FALSE
```

**Comment by thehighwayman** [comment]
you're right I'll do that

**Comment by Amelia0010** [comment]
why disable shadowpeople entirely and not just slings 🤔

File: yogstation/code/modules/mob/living/carbon/human/species_types/halloween.dm [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/datum/species/shadow/ling/check_roundstart_eligible()
```
Comment (Amelia0010) [inline yogstation/code/modules/mob/living/carbon/human/species_types/halloween.dm]: this should be moved to the sling species folder, which is in the sling gamemode folder

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: yogstation/code/modules/mob/living/carbon/human/species_types/halloween.dm (hunk 1 above)
Comment (thehighwayman) [reply yogstation/code/modules/mob/living/carbon/human/species_types/halloween.dm]: just this is more MODULAR

**Comment by Time-Green** [comment]
no bad 👎

**Comment by thehighwayman** [comment]
you're bad

File: yogstation/code/modules/mob/living/carbon/human/species_types/halloween.dm (hunk 1 above)
Comment (Amelia0010) [reply yogstation/code/modules/mob/living/carbon/human/species_types/halloween.dm]: it's not tho

**Comment by fluffe9911** [comment]
Did you stealth remove zombies and shadow man ling? lol

**Comment by Time-Green** [comment]
yeah he stealth removed zombies

by github law this pr must be reverted

**Pull request merged**